lib/go-tc/steering.go

```diff
@@ -19,18 +19,22 @@ package tc
  * under the License.
  */
 
+// Steering holds information about a steering delivery service.
 type Steering struct {
 	DeliveryService DeliveryServiceName      `json:"deliveryService"`
 	ClientSteering  bool                     `json:"clientSteering"`
 	Targets         []SteeringSteeringTarget `json:"targets"`
 	Filters         []SteeringFilter         `json:"filters"`
 }
 
+// SteeringFilter is a filter for a target delivery service.
 type SteeringFilter struct {
 	DeliveryService DeliveryServiceName `json:"deliveryService"`
 	Pattern         string              `json:"pattern"`
 }
 
+// SteeringSteeringTarget is a target delivery service of a steering delivery
+// service
```

Comment:
Missing period

Reply:
Addressed in 69a99ae175.

Reply:
[No it wasn't](https://github.com/apache/trafficcontrol/pull/4309/files#diff-aba1cae0f4f29324491134896613f975R36-R37)

Reply:
Addressed for real in 7f4d95bdaa.